contribs/gmf/src/services/objecteditingmanager.js

```diff
@@ -1,5 +1,148 @@
 goog.provide('gmf.ObjectEditingManager');
 
+goog.require('gmf.EditFeature');
+goog.require('ngeo.Location');
+goog.require('ol.Feature');
+
+
+/**
+ * A service that looks for certain parameters in the url and use them to fetch
+ * a feature using the GMF protocol.
+ *
+ * @param {angular.$q} $q Angular $q service.
+ * @param {gmf.EditFeature} gmfEditFeature Gmf edit feature service.
+ * @param {ngeo.Location} ngeoLocation ngeo location service.
+ * @constructor
+ * @struct
+ * @ngInject
+ */
+gmf.ObjectEditingManager = function($q, gmfEditFeature, ngeoLocation) {
+
+  /**
+   * @type {angular.$q}
+   * @private
+   */
+  this.q_ = $q;
+
+  /**
+   * @type {gmf.EditFeature}
+   * @private
+   */
+  this.gmfEditFeature_ = gmfEditFeature;
+
+  /**
+   * @type {ngeo.Location}
+   * @private
+   */
+  this.ngeoLocation_ = ngeoLocation;
+
+  /**
+   * @type {angular.$q.Deferred|null}
+   * @private
+   */
+  this.getFeatureDefered_ = null;
+
+};
+
+
+/**
+ * Use the EditFeature service to fetch a single feature using parameters in
+ * the url. The method returns a promise that has the feature as argument in
+ * the callback method. If any parameter in the url is missing, `null` is
+ * returned, otherwise the query is made. If the query returns a feature, it
+ * is returned, otherwise one is created with empty geometry and with the
+ * property set.
+ *
+ * @return {angular.$q.Promise} Promise.
+ * @export
+ */
+gmf.ObjectEditingManager.prototype.getFeature = function() {
+
+  if (!this.getFeatureDefered_) {
+    this.getFeatureDefered_ = this.q_.defer();
+
+    var geomType = this.ngeoLocation_.getParam(
+      gmf.ObjectEditingManager.Param.GEOM_TYPE);
+    var id = this.ngeoLocation_.getParam(
+      gmf.ObjectEditingManager.Param.ID);
+    var layer = this.ngeoLocation_.getParam(
+      gmf.ObjectEditingManager.Param.LAYER);
+    var property = this.ngeoLocation_.getParam(
+      gmf.ObjectEditingManager.Param.PROPERTY);
+    var theme = this.ngeoLocation_.getParam(
+      gmf.ObjectEditingManager.Param.THEME);
+
+    if (geomType && id && layer && property && theme) {
+      this.gmfEditFeature_.getFeaturesWithComparisonFilters(
+        [layer],
+        [{
+          operator: 'eq',
+          property: property,
+          value: id
+        }]
+      ).then(this.handleGetFeatures_.bind(this, property, id));
+    } else {
+      this.getFeatureDefered_.resolve(null);
+    }
+  }
+
+  return this.getFeatureDefered_.promise;
+
+};
+
+
+/**
+ * @return {?string} The geometry type.
```

Comment:
I got some error with the doc generation (not because of this line):
 1. Can you check the apidoc generation for this PR ?
 2. Can you check if our apidoc says that the return can be null here ? Otherwise, can you check this entire PR and add `or null` in the description of the returned value ?